### DISCUSSION TITLE
Can this be used as in qemu vnc server

### DISCUSSION THREAD

**Discussion by liyimeng** [any1/neatvnc#131] [General]
Hi @any1, is this possible to be integrated into qemu vnc server? I use qemu to run vm and noVNC to access it. I see you have enabled h.264 support in noVNC. It will be great if h.264 encoder is also enabled on qemu side. And I sure I am not the only one of the millions users, who will greatly appreciate your outstanding work!

**Comment by any1** [comment]
That seems like a question better aimed to qemu developers.

**Reply by liyimeng** [reply]
That is right. But I think you expertis in vnc would be great help here. I personally  consider doing something on this, but I lack of relevant knowledge to do something sensible. I am wondering if adding a file like this https://github.com/qemu/qemu/blob/master/ui/vnc-enc-tight.c is enough?

**Reply by any1** [reply]
I really think that you should ask the qemu developers. I am not familiar with that codebase.
